src/main/java/mil/dds/anet/views/AbstractCustomizableAnetBean.java

```diff
@@ -17,4 +24,15 @@ public String getCustomFields() {
   public void setCustomFields(String customFields) {
     this.customFields = Utils.trimStringReturnNull(customFields);
   }
+
+  public void checkAndFixCustomFields() {
+    try {
+      setCustomFields(Utils.sanitizeJSON(getCustomFields()));
+    } catch (JsonProcessingException e) {
+      setCustomFields(null);
+      logger.error("Unable to process Json, CustomFields payload discarded", e);
```

Comment:
Small suggestion: `"Unable to process Json, customFields payload discarded"` as the field is called `customFields` (with lowercase 'c') in the GraphQL schema.